t/020-fifottl.t

```diff
@@ -245,6 +245,22 @@ test:test('ttl after delay test', function(test)
     test:is(task.ttr, TTR * 1000000, 'check TTR after release')
 end)
 
+test:test('buried task in dropped queue', function(test)
```

Comment:
I would write it in simpler way:

```lua
-- gh-96: infinite loop after drop a tube with burried task
test:test('buried task in dropped queue', function(test)
    test:plan(1)

    local TASK_ID = 1
    local tube = queue.create_tube('test_drop_with_burried', 'fifottl',
        {ttr = 0.1, if_not_exist = true})

    tube:put({foo = 'bar'})
    local task = tube:take(0)
    tube:bury(task[TASK_ID])

    tube:drop()
    fiber.sleep(0.2)
    test:ok(true, 'queue does not hang')
end)
```

Reply:
This really is much simpler. We don't need 2 task to reproduce, that was a mistake.